conf/base.config

```diff
@@ -67,4 +62,7 @@ process {
         cpus   = { check_max( 4 * task.attempt, 'cpus' ) }
         memory = { check_max( 12.GB * task.attempt, 'memory' ) }
     }
+    withName:'FILTERCONSENSUSREADS' {
+        memory = { check_max( 12.GB * task.attempt, 'memory' ) }
```

Comment:
Gotcha.  Why not add the `error_retry` on the process definition as well?

Reply:
A few lines above there is:
```
    errorStrategy = { task.exitStatus in ((130..145) + 104) ? 'retry' : 'finish' }
    maxRetries    = 1
```
Which sets by default retrying for all processes.

Reply:
I asked one of the Seqera nf-core folks at ISMB where _module_-level config should live and they said not in `base.config` but in `modules.config`.  That's also my reading in the nf-core docs, as well as in other nf-core pipelines (e.g. `withName` are in the `modules.config` or `modules/<process>.config`, whereas `withLabel` are in the `base.config`). The [nf-core docs even say to copy the `modules.config`](https://nf-co.re/docs/usage/getting_started/configuration#understanding-and-modifying-tool-arguments). 

Can you please move it back to `modules.config`?